os/android/drmhwctwo.cpp

```diff
@@ -65,12 +84,15 @@ DrmHwcTwo::DrmHwcTwo() {
 }
 
 HWC2::Error DrmHwcTwo::Init() {
+  OS::GetExplicitSync();
+
   displays_.emplace(std::piecewise_construct,
                     std::forward_as_tuple(HWC_DISPLAY_PRIMARY),
                     std::forward_as_tuple(&device_, HWC_DISPLAY_PRIMARY,
                                           HWC2::DisplayType::Physical));
 
   displays_.at(HWC_DISPLAY_PRIMARY).Init();
```

Comment:
here in Init call you could query the run time setting and check if we need to disabled explicit sync for example